src/network/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const BLOCK_TOPIC = "beacon_block";
```

Comment:
Do we want to move these to global constants directory `constants/networking`

Reply:
Yea that works. The sync module will be using these constants, too.